### ISSUE TITLE
Setting to "Refresh & Scan" Artists automatically when performing a "Manual Import".

### ISSUE THREAD

**Issue by Simpuhl** [Lidarr/Lidarr#4151]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Let's say I've just submitted 10+ new albums to MusicBrainz and waited until after :05 for Lidarr Database (backend) to fresh that data. A manual Refresh & Scan must still be run for each Artist in order for it to import into the Lidarr (client) side. This can be very annoying.

### Describe the solution you'd like

An optional setting to enable Automatic Artist "Refresh & Scan" when performing a manual import.

Basically, after clicking manual import, any artists detected would first run a Refresh and Scan before attempting to detect the album.

### Describe alternatives you've considered

None

### Anything else?

No
